src/Tickets/Admin/All_Tickets/Screen_Options.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Class the handles the All Tickets screen options.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Admin\All_Tickets
+ */
+
+namespace TEC\Tickets\Admin\All_Tickets;
+
+/**
+ * Class Screen_Options
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Admin\All_Tickets
+ */
+class Screen_Options {
+	/**
+	 * @var string The user option that will store how many attendees should be shown per page.
+	 */
+	public static $per_page_user_option = 'event_tickets_all_tickets_per_page';
+
+	/**
+	 * @var string The screen id these screen options should render on.
+	 */
+	protected $screen_id;
```

Comment:
Do we use/need this? It should be accessible via `$this->screen->id` after all.

Reply:
Probably not. I think I may have lifted that idea from another file. I'll remove it. Thanks!

Reply:
We don't use `$this->screen` or `$this->screen_id` anyway. I removed both.